src/test/java/org/fcrepo/client/integration/FcrepoClientIT.java

```diff
@@ -108,6 +108,8 @@ public void testPostBinary() throws Exception {
                 CREATED.getStatusCode(), status);
         assertEquals("Location did not match slug", serverAddress + slug, response.getLocation().toString());
 
+        assertNotNull("Didn't find linked description!", response.getLinkHeaders("describedby").get(0));
```

Comment:
The Link header returned when creating a binary has three parts: `Link: <description>, rel=describedby; anchor=<binary>`

This test verifies that the existing situation of having a response with a three-part Link header can now be parsed correctly, instead of resulting in a NullPointerException (the previous behavior).